core-web/libs/edit-content/src/lib/fields/dot-edit-content-radio-field/dot-edit-content-radio-field.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="field-checkbox" *ngFor="let option of options">
+    <p-radioButton
+        [inputId]="option.value"
```

Comment:
We can't prevent the user to have the same value twice which result in this:

<img width="127" alt="image" src="https://github.com/dotCMS/core/assets/751424/1b64644f-7bc0-4197-b96f-586bcfda2dab">

We can use `field.variable` which is unique and the index of the `ngFor` to create a unique id.